py2exe_setuptools.py

```diff
@@ -113,6 +113,13 @@ def build_interp(self, ext):
         else:
             ext_path += ".dll"
 
+        if 'GCC' in sys.version:
+            if ext.name == "py2exe.run_ctypes_dll":
```

Comment:
I add this fix because of this error:
```
writing build/temp.mingw_x86_64_clang-cpython-311/source/run_ctypes_dll-py311-mingw_x86_64_clang.def
clang -shared -Wl,--enable-auto-image-base -s build/temp.mingw_x86_64_clang-cpython-311/source/run_ctypes_dll.o build/temp.mingw_x86_64_clang-cpytho
n-311/source/start.o build/temp.mingw_x86_64_clang-cpython-311/source/icon.rc.o build/temp.mingw_x86_64_clang-cpython-311/source/memorymodule.o buil
d/temp.mingw_x86_64_clang-cpython-311/source/myloadlibrary.o build/temp.mingw_x86_64_clang-cpython-311/source/_memimporter.o build/temp.mingw_x86_64
_clang-cpython-311/source/actctx.o build/temp.mingw_x86_64_clang-cpython-311/source/python-dynload.o build/temp.mingw_x86_64_clang-cpython-311/sourc
e/run_ctypes_dll-py311-mingw_x86_64_clang.def -LC:/MSYS64/clang64/lib -luser32 -lshell32 -lpython3.11 -o build/lib.mingw_x86_64_clang-cpython-311/py
2exe/run_ctypes_dll-py311-mingw_x86_64_clang.dll -m64 -mwindows
ld.lld: error: unknown directive: ,
clang: error: linker command failed with exit code 1 (use -v to see invocation)
error: command 'C:\\MSYS64\\clang64\\bin/clang.EXE' failed with exit code 1

ERROR Backend subprocess exited when trying to invoke build_wheel
```

Reply:
Ok I guess I will remove this once I remove the whole `run_ctypes_dll` target.